lib/lte_link_control/lte_lc.c

```diff
@@ -346,6 +686,35 @@ int lte_lc_init_and_connect(void)
 	return err;
 }
 
+int lte_lc_connect_async(lte_lc_evt_handler_t handler)
+{
+	if (handler) {
+		evt_handler = handler;
+	} else if (evt_handler == NULL) {
+		LOG_WRN("No handler registered, events will not be received");
+	}
+
+	return w_lte_lc_connect(false);
+}
+
+int lte_lc_init_and_connect_async(lte_lc_evt_handler_t handler)
+{
+	int err;
+
+	if (handler) {
+		evt_handler = handler;
+	} else if (evt_handler == NULL) {
+		LOG_WRN("No handler registered, events will not be received");
```

Comment:
Same question as before: how do we know when we are connected if we cannot receive events?

Reply:
Solved same way as above